kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.ui.xml

```diff
@@ -37,7 +37,7 @@
     </ui:style>
 
 
-    <b:Container fluid="true" addStyleNames="packages-panel">
+    <b:Container fluid="true">
```

Comment:
Please change the header to: "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"